api/tests/routes/saml/educonnect_test.py

```diff
@@ -516,6 +518,29 @@ def test_educonnect_connection_updates_validated_birth_date(self, mock_get_educo
         client.post("/saml/acs", form={"SAMLResponse": "encrypted_data"})
         assert user.validatedBirthDate == birth_date
 
+    @patch("pcapi.connectors.beneficiaries.educonnect.educonnect_connector.get_educonnect_user")
+    def test_educonnect_error_is_tracked(self, mock_get_educonnect_user, client, app):
+        user, request_id = self.connect_to_educonnect(client, app)
+        age = 20
+        mock_get_educonnect_user.return_value = users_factories.EduconnectUserFactory(
+            saml_request_id=request_id, age=age
+        )
```

Comment:
(détails) On n'a pas vraiment besoin de déclarer `age` : 

```suggestion
        mock_get_educonnect_user.return_value = users_factories.EduconnectUserFactory(
            saml_request_id=request_id, age=20
        )
```